<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>MainWindow</class>
	<widget class="QMainWindow" name="MainWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>800</width>
				<height>600</height>
				</rect>
			</property>
		<property name="windowTitle">
			<string>MainWindow</string>
			</property>
		<widget class="QWidget" name="centralwidget">
			<layout class="QVBoxLayout" name="verticalLayout">
				<item>
					<widget class="QLabel" name="totalAmountLabel">
						<property name="font">
							<font>
								<pointsize>20</pointsize>
								<bold>true</bold>
								</font>
							</property>
						<property name="text">
							<string>0.0</string>
							</property>
						<property name="indent">
							<number>20</number>
							</property>
						</widget>
					</item>
				<item>
					<widget class="QTableView" name="tableView">
						<property name="font">
							<font>
								<pointsize>12</pointsize>
								</font>
							</property>
						</widget>
					</item>
				<item>
					<layout class="QHBoxLayout" name="horizontalLayout_2">
						<item>
							<widget class="QPushButton" name="addButton">
								<property name="font">
									<font>
										<pointsize>12</pointsize>
										</font>
									</property>
								<property name="text">
									<string>Add</string>
									</property>
								</widget>
							</item>
						<item>
							<widget class="QPushButton" name="editButton">
								<property name="font">
									<font>
										<pointsize>12</pointsize>
										</font>
									</property>
								<property name="text">
									<string>Edit</string>
									</property>
								</widget>
							</item>
						<item>
							<widget class="QPushButton" name="removeButton">
								<property name="font">
									<font>
										<pointsize>12</pointsize>
										</font>
									</property>
								<property name="text">
									<string>Remove</string>
									</property>
								</widget>
							</item>
						</layout>
					</item>
				</layout>
			</widget>
		</widget>
	<resources/>
	<connections/>
	</ui>
